src/NewTwitchApi/Resources/EntitlementsApi.php

```diff
@@ -20,4 +21,38 @@ public function getEntitlementGrantsUploadURL(string $bearer, string $manifestId
 
         return $this->postApi('entitlements/upload', $bearer, $queryParamsMap);
     }
+
+    /**
+     * @throws GuzzleException
+     * @link https://dev.twitch.tv/docs/api/reference#get-code-status
+     */
+    public function getStatusCode(string $bearer, array $codes = [], int $userId): ResponseInterface
```

Comment:
Required arguments should come before optional arguments. Also, shouldn't this be `getCodeStatus()`?